GOT-StudentVersion/GOT-StudentVersion/DetailViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  DetailViewController.swift
+//  GOT-StudentVersion
+//
+//  Created by Kevin Natera on 8/13/19.
+//  Copyright © 2019 C4Q . All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class DetailViewController : UIViewController {
+    
+    var gotep: GOTEpisode!
+    
+    
+    @IBOutlet weak var gotImageOutlet: UIImageView!
+    
+    @IBOutlet weak var seasonLabel: UILabel!
+    
+    @IBOutlet weak var episodeLabel: UILabel!
+    
+    @IBOutlet weak var runtimeLabel: UILabel!
+    
+    @IBOutlet weak var airdateLabel: UILabel!
+    
+    @IBOutlet weak var descriptionOutlet: UITextView!
+    
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        gotImageOutlet.image = UIImage(named: "\(gotep.originalImageID)")
```

Comment:
This is great, but wrap all of this code into a private func and then call it in the viewDidLoad(). It makes everything cleaner and separates out tasks